<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="ru.stqa.pft.adressbook.tests.ContactCreationTests" tests="3" skipped="0" failures="0" errors="0" timestamp="2018-06-20T18:29:54" hostname="DESKTOP-UT1S4OG" time="29.463">
  <properties/>
  <testcase name="testBadContactCreation" classname="ru.stqa.pft.adressbook.tests.ContactCreationTests" time="13.829"/>
  <testcase name="testСontactContact[0](ContactData{id='2147483647', name1='Gabdullin 0', name2='Almaz 0'})" classname="ru.stqa.pft.adressbook.tests.ContactCreationTests" time="7.782"/>
  <testcase name="testСontactContact[1](ContactData{id='2147483647', name1='Gabdullin 1', name2='Almaz 1'})" classname="ru.stqa.pft.adressbook.tests.ContactCreationTests" time="7.852"/>
  <system-out><![CDATA[21:29:54.484 [Test worker] INFO  r.stqa.pft.adressbook.tests.TestBase - Start test testBadContactCreation with parametrs []
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
